### PR TITLE
recette des crepes #3

### PR DESCRIPTION
**Pull request by baaluidnrey**
Hi, this is my recipe of the "crêpes" !
Please give me your feedback !

### CODE REVIEW

File: desserts/crepes.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Comment faire de bonnes crêpes ?
+
+## Les ingrédients (pour environ 20 crêpes, mais je ne suis pas sure)
+
+- 300g de farine
+- 3 oeufs
+- beurre (50g)
+- lait (environ 1L mais c'est adapté à la fin)
```
Comment (SamHadjes) [inline desserts/crepes.md:8]: the order <amount> <ingredient> should be kept on all lines

File: desserts/crepes.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+- Mettre les oeufs dans le puit
+- Commencer à mélanger doucement et ajoutant régulièrement du lait
+- Quand le mélange est fluide, ajouter la seconde moitié de la farine en mélangeant
+- Ajouter le beurre fondu et du lait en fonction de la texture (on doit obtenir à la fin un mélange fluide et liquide (mais pas trop, le "pas trop" se vérifiera à la cuisson des premières crêpes communément appelées "premières crêpes ratées")
+- AJouter du rhum, de la bière ou autre en fonction de ce qui vous plaît
+- Laisser reposer une heure
+
+## La cuisson
+
+- Le mélange doit être liquide (mais pas trop), c'est-à-dire que l'on doit être capable de l'étaler dans la poêle en moins de 5 secondes (ce sera vérifié à la première cuisson puis adapté. Si la pâte s'étale mal, il faut ajouter du lait)
```
Comment (SamHadjes) [inline desserts/crepes.md:24]: should we add a chronometer in the list of needed tools for the recipe?

**Review by SamHadjes — CHANGES_REQUESTED** [review]
Mostly ok, some changes on ingredients and one question about needed tools

File: desserts/crepes.md (hunk 1 above)
Comment (baaluidnrey) [reply desserts/crepes.md]: ok I change it, thank you

**Comment by baaluidnrey** [comment]
I've changed the code a bit, is it all right for you now ?

File: desserts/crepes.md [hunk 3]
```diff
@@ -4,11 +4,19 @@
...
-- beurre (50g)
-- lait (environ 1L mais c'est adapté à la fin)
+- 50g de beurre
+- 1L de lait (environ, on adapte à la cuisson)
 - pincée de sel
 - du rhum, si vous en avez
 
+## Les ustensiles
+
+- une grande jatte
```
Comment (SamHadjes) [inline desserts/crepes.md:14]:
```suggestion
- une grande jatte ou un grand saladier
```

**Review by SamHadjes — CHANGES_REQUESTED** [review]
Almost done

**Review by SamHadjes — APPROVED** [review]
Everything good

**Pull request merged**